content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -157,116 +191,293 @@ Automatic reordering rules, enabled by setting the reordering rule's :guilabel:`
+forecasted quantity falls below a specified minimum. Products on this dashboard are called *needs*,
+because they are needed to fulfill upcoming sales orders, for which the forecasted quantity is not
+enough.
 
 The replenishment dashboard, accessible by navigating to :menuselection:`Inventory app -->
 Operations --> Replenishment`, considers sales order deadlines, forecasted stock levels, and vendor
-lead times. It displays needs **only** when it is time to reorder items.
-
-.. note::
-   If the one-day window for ordering products is too short, skip to the :ref:`visibility days
-   <inventory/product_management/visibility-days>` section to make the need appear on the
-   replenishment dashboard a specified number of days in advance.
+lead times. It displays needs **only** when it is time to reorder items, thanks to the :guilabel:`To
+Reorder` filter.
 
 .. image:: reordering_rules/manual.png
    :align: center
    :alt: Click the Order Once button on the replenishment dashboard to replenish stock.
 
-.. _inventory/product_management/visibility-days:
+.. _inventory/warehouses_storage/route:
 
-Visibility days
+Preferred route
 ===============
 
+Odoo allows for multiple routes to be selected as replenishment methods under the
+:guilabel:`Inventory` tab on each product form. For instance, it is possible to select both
+:guilabel:`Buy` and :guilabel:`Manufacture`, indicating to Odoo that the product can be bought or manufactured.
+
+Odoo also enables users to set a preferred route for a product's reordering rule. This is the route
+that the rule defaults to if multiple are selected. To select a preferred route, begin by navigating
+to :menuselection:`Inventory app --> Configuration --> Reordering Rules` or
+:menuselection:`Inventory app --> Operations --> Replenishment`.
+
+Click inside of the column on the row of a reordering rule, and a drop-down menu shows all available
+routes for that rule. Select one to set it as the preferred route.
+
+.. image:: reordering_rules/select-preferred-route.png
+   :alt: Select a preferred route from the drop-down.
+
 .. important::
-   Ensure :doc:`lead times <lead_times>` are understood before proceeding with this section.
+   If multiple routes are enabled for a product but no preferred route is set for its reordering
+   rule, the product is reordered using the selected route that is listed first on the
+   :guilabel:`Inventory` tab of the product form.
+
+Advanced uses
+-------------
+
+Pairing :guilabel:`Preferred Route` with one of the following fields on the replenishment report
+unlocks advanced configurations of reordering rules. Consider the following:
+
+.. _inventory/warehouses_storage/set-vendor:
+
+- :guilabel:`Vendor`: When the selected :guilabel:`Preferred Route` is :guilabel:`Buy`, setting the
+  :guilabel:`Vendor` field to one of the multiple vendors on the vendor pricelist indicates to Odoo
+  that is the vendor to prioritize when making orders.
```

Comment:
Be more explicit - if that is the vendor to prioritize, does Odoo automatically populate this vendor on POs then?